posthog/models/feature_flag/flag_matching.py

```diff
@@ -44,6 +45,8 @@
     labelnames=["reason"],
 )
 
+DATABASE_FOR_FLAG_MATCHING = "default" if "decide" not in settings.READ_REPLICA_OPT_IN else "replica"
```

Comment:
I'm guessing moving this to settings will remove and cyclic dependency issues if there were any?